cmd/publicize/server.go

```diff
@@ -0,0 +1,188 @@
+package main
+
+import (
+	"fmt"
+	"net/url"
+	"path"
+	"regexp"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	prowconfig "k8s.io/test-infra/prow/config"
+	"k8s.io/test-infra/prow/git/v2"
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/pluginhelp"
+)
+
+type githubClient interface {
+	IsMember(org, user string) (bool, error)
+	CreateComment(owner, repo string, number int, comment string) error
+	GetPullRequest(org, repo string, number int) (*github.PullRequest, error)
+
+	ListIssueComments(owner, repo string, number int) ([]github.IssueComment, error)
+}
+
+var publicizeRe = regexp.MustCompile(`(?mi)^/publicize\s*$`)
+
+func helpProvider(_ []prowconfig.OrgRepo) (*pluginhelp.PluginHelp, error) {
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: `The publicize plugin is used for merging a pull request into the bonded repository.`,
```

Comment:
Bonded is not a word people use in this context. Let's just call it the upstream.